libs/platform/src/lib/components/form/input/input.component.ts

```diff
@@ -39,15 +42,42 @@ export type InputType = 'text' | 'number' | 'email';
     selector: 'fdp-input',
     templateUrl: 'input.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush,
+    styleUrls: ['input.component.scss'],
     providers: [
-        { provide: FormFieldControl, useExisting: InputComponent, multi: true }
+        { provide: FormFieldControl, useExisting: PlatformInputComponent, multi: true }
     ]
 })
-export class InputComponent extends BaseInput {
+export class PlatformInputComponent extends BaseInput implements CssClassBuilder {
```

Comment:
Please stop this renaming, Checkout other platform components, yout prefix component, we prefix only module. the same for files names